### PR TITLE
ENH: add support for WPS3.8

### PR DESCRIPTION
**Pull request by andreas-h**
- fix broken download link (l. 69)
- replace geodata download with larger file which also contains GMTED2010 topography which is needed by WPS3.8 (l. 70) see http://www2.mmm.ucar.edu/wrf/users/download/get_sources_wps_geog.html
- support `ungrib` testing for WPS3.8, which doesn't contain `Vtable.ARW` any more. However, when using `Vtable.ARW.UPP`, the test in WPS3.8 succeeds.

### CODE REVIEW

**Comment by andreas-h** [comment]
any news on this one? can i do anything else to get this merged into master?

**Comment by boegel** [comment]
@andreas-h Thanks for the bump, I'll try and take a look at this ASAP.

I changed the target branch to `develop`, since the only time we merge PRs directly into `master` is when we do a new EasyBuild release (cfr. #999).

**Comment by boegel** [comment]
@andreas-h apparently there's a merge conflict with current `develop` and your branch...

Are you up for fixing that? If you're not sure how, I can try and help out myself and issue a PR to your `wps38` branch to fix that.

**Comment by boegel** [comment]
@andreas-h merge conflict fixed with https://github.com/andreas-h/easybuild-easyblocks/pull/2, please merge?

**Comment by boegel** [comment]
@andreas-h made some minor enhancements in https://github.com/andreas-h/easybuild-easyblocks/pull/3, please review

File: easybuild/easyblocks/w/wps.py [hunk 1]
```diff
@@ -67,8 +67,8 @@ def __init__(self, *args, **kwargs):
     @staticmethod
     def extra_options():
         testdata_urls = [
-            "http://www.mmm.ucar.edu/wrf/src/data/avn_data.tar.gz",
-            "http://www.mmm.ucar.edu/wrf/src/wps_files/geog.tar.gz",  # 697MB download, 16GB unpacked!
+            "http://www2.mmm.ucar.edu/wrf/src/data/avn_data.tar.gz",
+            "http://www2.mmm.ucar.edu/wrf/src/wps_files/geog_complete.tar.bz2",  # 2.3GB download!
```
Comment (andreas-h) [inline easybuild/easyblocks/w/wps.py]: @boegel Trying to do this, I noticed that the URLs are defined in `extra_options`, which is a static method and thus doesn't have access to `self.version`. I'd appreciate any suggestions how to solve this.

Comment (boegel) [inline easybuild/easyblocks/w/wps.py]: @andreas-h then we'll need to define `None` as default version for the URLs, and initialise it properly in the easyblock constructor (`__init__`) based on `self.version`

**Comment by boegel** [comment]
@andreas-h The diff for this PR is all messed up now, what happened? :)

**Comment by andreas-h** [comment]
interesting ... what I did was

    git fetch upstream
    git rebase upstream/develop
    git push

should I try to fix this (how?) or just close and open a new one?

**Comment by boegel** [comment]
It seems like GitHub isn't digesting the `rebase` very well... The downside of a rebase is that you're rewriting history, which can mess things up. I usually just do a `git merge` of `develop`.

I'm not sure if this PR can still be salvaged (maybe with a `merge develop`?), just closing it and opening a new one is fine by me (just include a reference to this one in the description)...

**Comment by andreas-h** [comment]
closing in favour of #1078

**Pull request closed without merging**